### ISSUE TITLE
A.I. Nodes Limited To 4096

### ISSUE THREAD

**Issue by CrunchySoap** [Facepunch/garrysmod-requests#1925]
This seemed like more of a bug than a feature request, so here we are.

Anyway, I was creating the nodegraph for a large map when I ended up hitting the limit, 4096. I spent a couple days trying to figure out how to increase this, but it appears to be hard-wired into GMod and well beyond my understanding. If this could be increased by a couple thousand, I could finally complete the A.I. Nodes. Please and thank you! Cheers.

**Comment by Grocel** [comment]
It used to be like 1500. Good to know that apparently it is 4096 nowadays. The limit was/is actually in-engine and not in map compile.

**Comment by robotboy655** [comment]
The limit is 8192 now.

**Comment by Grocel** [comment]
> The limit is 8192 now.

Was this actually changed just now? There isn't anything about this in comments of the past 7 days.

**Comment by robotboy655** [comment]
No. It's not recent, https://github.com/Facepunch/garrysmod-requests/issues/2786
